patroni/ha.py

```diff
@@ -67,11 +67,14 @@ def touch_member(self):
         self.dcs.touch_member(json.dumps(data, separators=(',', ':')))
 
     def clone(self, clone_member=None, msg='(without leader)'):
-        if self.state_handler.bootstrap(cluster_initialized=True, clone_member=clone_member):
+        if self.state_handler.clone(clone_member):
             logger.info('bootstrapped %s', msg)
+            node_to_follow = self._get_node_to_follow(self.dcs.get_cluster())
+            if node_to_follow and node_to_follow.name == self.state_handler.name:
```

Comment:
Should we do the check for node_to_follow.name == self.state_handler.name inside the _get_node_to_follow and just return None in this case?